patches/tModLoader/Terraria/Localization/Content/ru-RU/tModLoader.json

```
@@ -3,352 +3,353 @@
+		"LoadErrorMajorVersionMismatch": "{0} поддерживает {2} версии {1}, но у вас установлено новое крупное обновление с версией ({3}), которая может быть несовместима. {0} необходимо обновить.",
+		"LoadWarningBannerOrBannerItemNotSet": "{0}: {1}. Вы объявили свойство ModNPC.banner или ModNPC.bannerItem, но не указали один из них. Убедитесь, что вы указали оба в своём коде: проверьте правописание и убедитесь, что предметы существуют, когда вы используете mod.ItemType(). Сообщите об этой ошибке разработчикам мода {0}.",
+		"UnloadError": "Во время выгрузки модов возникла одна или несколько ошибок. Во избежание дальнейших проблем необходимо перезапустить tModLoader.",
+		"DefensiveUnload": "Мод {0} не смог правильно выгрузиться. Разработчики модов должны использовать безопасное программирование для гарантии завершения выгрузки даже в случае ошибок во время их загрузки.",
+		"BuildError": "Во время сборки мода {0} произошла ошибка",
+		"CompileError": "Компиляция мода {0} провалилась с {1} {^1:ошибкой;ошибками} и {2} {^2:предупреждением;предупреждениями}.",
+		"BuildErrorModNameEmpty": "У модов не может быть пустое название",
+		"BuildErrorModNamedTerraria": "Моды не могут иметь название Terraria",
+		"BuildErrorModNameHasPeriod": "Внутреннее название мода не может содержать символ точки",
+		"BuildErrorTwoModsSameName": "У двух разных модов обнаружено одно и то же внутреннее имя {0}",
+		"BuildErrorModReference": "Не удалось прочесть ссылку на мод {0}",
+		"BuildErrorMissingNoCompile": "У вас отсутствуют dll файлы для параметра noCompile: {0} или {1}\n\nОбратите внимание, что All.dll не должен содержать ссылок на Microsoft.Xna.Framework или FNA\nXNA.dll должен ссылаться на правильный tModLoader.exe и фреймворк.",
+		"BuildErrorModNameDoesntMatchAssemblyName": "Название мода {0} не совпадает с названием сборки {1}",
+		"BuildErrorNamespaceFolderDontMatch": "Пространство имён и название папки не совпадают. Пространство имён верхнего уровня должно совпадать с названием папки.",
+		"BuildErrorNoModClass": "Убедитесь, что в вашем проекте мода присутствует только один класс, наследующийся от класса Mod.",
+		"BuildErrorLoadingPrecompiled": "Не удалось загрузить заранее скомпилированную библиотеку {0}",
 		"BuildErrorFailedLoadBuildTxt": "Не удалось загрузить {0}",
-		"BuildWarningEACWindowsOnly": "Edit and Continue поддерживается только на Windows.",
-		"LoadErrorTextureFailedToLoad": "Не удалось загрузить файл текстур {0}",
-		"LoadErrorWavSoundFailedToLoad": "Не удалось загрузить Wav аудио файл {0}",
-		"LoadErrorMP3SoundFailedToLoad": "Не удалось загрузить MP3 аудио файл {0}",
-		"LoadErrorSoundFailedToLoadAudioDeviceHint": "(Удостоверьтесь в том, что к вашему компьютеру подключены колонки или наушники, и в том что нужные устройства включены в панели управления.)",
+		"BuildWarningEACWindowsOnly": "Edit и Continue поддерживаются только на Windows.",
```

EaC is the name of a singular Development Productivity feature from .NET.
```suggestion
		"BuildWarningEACWindowsOnly": "Функция 'Edit and Continue' поддерживается только на Windows.",
```